src/utils/RuntimeHelper.js

```diff
@@ -16,35 +16,83 @@ limitations under the License.
 
 // @flow
 import _ from 'lodash'
-import runtimes from './runtimes'
+import Api from 'utils/Api'
+import { initialState } from 'store/clusters'
 
-export default class RuntimeHelper {
+const namespace = _.get(process.env, 'KUBELESS_NAMESPACE', 'kubeless')
```

Comment:
so kubeless could be deployed in a different namespace and the UI would pick that up if the env var is set properly ?

Reply:
yes, that's the idea